### PR TITLE
Add 4 new icons

### PR DESCRIPTION
**Pull request by naveenccmsd**
## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark:  Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* Bookmyshow (`com.bt.bms.id`) - added from icon request form
* Nova launcher (`com.teslacoilsw.launcher.prime`)
* OLA Cabs (`com.olacabs.customer`)
* Twelve club (`com.twelve.club`)

### Icons updated
*  OLX  (`com.olx.olx`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -2,119 +2,76 @@
...
-  <icon drawable="@drawable/termux" package="com.termux.window" name="Termux"/>
-  <icon drawable="@drawable/teufel" package="com.raumfeld.android.controller" name="Teufel Raumfeld"/>
   <icon drawable="@drawable/teufel" package="com.wifiaudio.Teufel" name="Teufel HOLIST"/>
   <icon drawable="@drawable/teufel" package="de.teufel.android.app.bluetooth.playstore" name="Teufel Headphones"/>
   <icon drawable="@drawable/tinkoff_work" package="ru.tinkoff.twork" name="TWork"/>
   <icon drawable="@drawable/tipster" package="tipster.evolutioit.app" name="Tipster"/>
-  <icon drawable="@drawable/tracker_control" package="net.kollnig.missioncontrol" name="Tracker Control"/>
   <icon drawable="@drawable/tricount" package="com.trilab.tricount.android" name="Tricount"/>
-  <icon drawable="@drawable/uniciv" package="com.unciv.app" name="Unciv"/>
-  <icon drawable="@drawable/vacation_days" package="rocks.poopjournal.vacationdays" name="Vacation Days"/>
```
Comment (naveenccmsd) [inline app/src/main/res/xml/grayscale_icon_map.xml:109]: vacation_days drawbales not found in scg folder

File: app/assets/appfilter.xml [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
   <calendar component="ComponentInfo{com.xiaomi.calendar/com.android.calendar.homepage.AllInOneActivity}" prefix="themed_icon_calendar_" name="Calendar"/>
+  <calendar component="ComponentInfo{foundation.e.calendar/com.android.calendar.AllInOneActivity}" prefix="themed_icon_calendar_" name="Calendar"/>
   <calendar component="ComponentInfo{org.calyxos.etar/com.android.calendar.AllInOneActivity}" prefix="themed_icon_calendar_" name="Calendar"/>
-  <calendar component="ComponentInfo{org.withouthat.acalendarplus/org.withouthat.acalendar.ACalendar}" prefix="themed_icon_calendar_" name="Calendar"/>
+  <calendar component="ComponentInfo{org.lineageos.etar/com.android.calendar.AllInOneActivity}" prefix="themed_icon_calendar_" name="Calendar"/>
+  <calendar component="ComponentInfo{org.sufficientlysecure.localcalendar/org.sufficientlysecure.localcalendar.ui.MainActivity}" prefix="themed_icon_calendar_" name="Calendar"/>
   <calendar component="ComponentInfo{org.withouthat.acalendar/org.withouthat.acalendar.ACalendar}" prefix="themed_icon_calendar_" name="Calendar"/>
```
Comment (naveenccmsd) [inline app/assets/appfilter.xml:18]: Ordred calendar in albpabetical based on drawable+package name

File: app/assets/appfilter.xml [hunk 3]
```diff
@@ -37,32 +37,24 @@
...
-  <item component="ComponentInfo{ai.asleep.android_aaron/ai.asleep.android_aaron.presentation.splash.SplashActivity}" drawable="generic_moon" name="Sleep Tracker"/>
-	<item component="ComponentInfo{com.dealwatch24.sleepcalculator/com.dealwatch24.sleepcalculator.MainActivity}" drawable="generic_moon" name="Sleep Tracker"/>
-	<item component="ComponentInfo{com.sleepmonitor.aio/com.sleepmonitor.aio.activity.SplashActivity}" drawable="generic_moon" name="Sleep Tracker"/>
-	<item component="ComponentInfo{ru.olegfilimonov.sleeptime/ru.olegfilimonov.sleeptime.activity.MainActivity}" drawable="generic_moon" name="Sleep Tracker"/>
-	<item component="ComponentInfo{com.sleep.snore.insomnia/com.sleepmonitor.aio.SplashActivity}" drawable="generic_moon" name="Sleep Tracker"/>
-	<item component="ComponentInfo{sleeptrakcer.sleeprecorder.sleepapp.sleep/sleeptrakcer.sleeprecorder.sleepapp.sleep.guide.SplashActivity}" drawable="generic_moon" name="Sleep Tracker"/>
-	<item component="ComponentInfo{com.fullpower.applications.sleepz/com.fullpower.applications.sleepz.SplashActivity}" drawable="generic_moon" name="Sleep Tracker"/>
```
Comment (naveenccmsd) [inline app/assets/appfilter.xml:47]: Added all above entry to below to keep alphabetical order

File: app/assets/appfilter.xml [hunk 4]
```diff
@@ -37,32 +37,24 @@
...
-  <item component="ComponentInfo{com.adidas.app/com.adidas.intro.IntroActivity}" drawable="adidas" name="adidas"/>
+  <item component="ComponentInfo{org.adaway/org.adaway.ui.home.HomeActivity}" drawable="adaway" name="AdAway"/>
   <item component="ComponentInfo{com.adidas.app/com.adidas.app.intro.IntroActivity}" drawable="adidas" name="adidas"/>
+  <item component="ComponentInfo{com.adidas.app/com.adidas.intro.IntroActivity}" drawable="adidas" name="adidas"/>
   <item component="ComponentInfo{com.adobe.reader/com.adobe.reader.AdobeReader}" drawable="adobe" name="Adobe Acrobat Reader"/>
   <item component="ComponentInfo{com.adobe.scan.android/com.adobe.scan.android.SplashActivity}" drawable="adobe_scan" name="Adobe Scan"/>
   <item component="ComponentInfo{com.adobe.scan.android/sample.example.dhruv.sample_apk.MainActivity}" drawable="adobe_scan" name="Adobe Scan"/>
-  <item component="ComponentInfo{com.dv.adm/com.dv.adm.Main}" drawable="advanced_download_manager" name="Advanced Download Manager"/>
-  <item component="ComponentInfo{com.dv.adm/com.dv.get.Main}" drawable="advanced_download_manager" name="Advanced Download Manager"/>
```
Comment (naveenccmsd) [inline app/assets/appfilter.xml:62]: removed duplicate entry

File: app/assets/appfilter.xml [hunk 5]
```diff
@@ -3809,35 +3994,34 @@
...
   <item component="ComponentInfo{com.xiaomi.hm.health/com.xiaomi.hm.health.activity.StartUpActivity}" drawable="zepp_life" name="Zepp Life"/>
-  <item component="ComponentInfo{com.xiaomi.hm.health/cn.com.smartdevices.bracelet.activity.MainActivity}" drawable="zepp_life" name="Zepp Life"/>
   <item component="ComponentInfo{com.zeptoconsumerapp/com.zeptoconsumerapp.MainActivity}" drawable="zepto" name="Zepto"/>
-  <item component="ComponentInfo{com.application.zomato/com.application.zomato.activities.Splash}" drawable="zomoto" name="Zomoto"/>
+  <item component="ComponentInfo{com.zeus.app/rideatom.app.features.SecondActivity}" drawable="zeus" name="ZEUS Deutschland"/>
   <item component="ComponentInfo{com.application.zomato/com.application.zomato.Splash}" drawable="zomoto" name="Zomoto"/>
+  <item component="ComponentInfo{com.application.zomato/com.application.zomato.activities.Splash}" drawable="zomoto" name="Zomoto"/>
   <item component="ComponentInfo{us.zoom.videomeetings/com.zipow.videobox.LauncherActivity}" drawable="zoom" name="Zoom"/>
-  <item component="ComponentInfo{com.jago.digitalBanking/com.jago.digitalBanking.MainActivity}" drawable="bank_jago" name="Jago"/>
```
Comment (naveenccmsd) [inline app/assets/appfilter.xml:3842]: Added to top of the file as per oeder

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 6]
```diff
@@ -2,119 +2,76 @@
...
   <icon drawable="@drawable/animiru" package="xyz.Quickdev.Animiru.mi" name="Animiru"/>
   <icon drawable="@drawable/bandbond" package="se.droid.bandbond" name="Bandbond"/>
-  <icon drawable="@drawable/bergfex" package="com.bergfex.mobile.android" name="bergfex"/>
   <icon drawable="@drawable/bitrix24" package="com.bitrix24.android" name="Bitrix24"/>
-  <icon drawable="@drawable/blindmate" package="eu.appiphany.blindmate" name="blindmate"/>
   <icon drawable="@drawable/camera" package="com.google.android.GoogleCamera.CamFour" name="Camera"/>
   <icon drawable="@drawable/camera_date_folders" package="de.kromke.andreas.cameradatefolders" name="Camera Date Folders"/>
   <icon drawable="@drawable/coinvero" package="com.candlestick_app" name="Coinvero"/>
   <icon drawable="@drawable/combined" package="de.peterge.combined" name="Combined!"/>
-  <icon drawable="@drawable/cookbook" package="de.uni_potsdam.hpi.openmensa" name="OpenMensa"/>
```
Comment (naveenccmsd) [inline app/src/main/res/xml/grayscale_icon_map.xml:15]: No svg present so removed it

Comment (Chefski) [reply app/src/main/res/xml/grayscale_icon_map.xml:15]: `cookbook.svg` was renamed to `generic_silverware`. It can be added back with the other drawable

**Review by Chefski — CHANGES_REQUESTED** [review]
Mind just making the 'O' in `olx.svg` a circle/oval (depending whether the current logo is fully circular or has a bit of an oval shape to it)? It doesn't seem to perfectly round
image

**Comment by naveenccmsd** [comment]
> Mind just making the 'O' in `olx.svg` a circle/oval (depending whether the current logo is fully circular or has a bit of an oval shape to it)? It doesn't seem to perfectly round image

It was intenationa to not to be acurate circle shape as the officalical logo is not circle.

image


Please review and let me know if you still the changes should be applied

**Comment by Chefski** [comment]
> It was intenationa to not to be acurate circle shape as the officalical logo is not circle.
> 
> image
> 
> Please review and let me know if you still the changes should be applied

`ola_cabs.svg` is fine. I was talking about `olx.svg`

image

**Review by Chefski — COMMENTED** [review]
SVG is now 160x160px. Wanna fix real quick and should be all
image

File: app/src/main/res/xml/grayscale_icon_map.xml (hunk 6 above)
Comment (naveenccmsd) [reply app/src/main/res/xml/grayscale_icon_map.xml:15]: Add generic_silverware insted of coobbook drawbles and moved to appfilter

https://github.com/LawnchairLauncher/lawnicons/pull/1014/commits/9e6629da3297d1edbcb0f65319e7e10ecb14ddac#diff-78a61c73b85c0c3925b77023f7eb3e58598e746095876a601b21c72023c9a846R651

**Comment by naveenccmsd** [comment]
> SVG is now 160x160px. Wanna fix real quick and should be all image

I used to work on 160x160 for better alignmenet like, align centre,top bottom . Forgot to change the size. I have updated now.

**Comment by Chefski** [comment]
Thanks!

**Pull request merged**